.storybook/main.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  stories: ["../src/**/*.stories.@(tsx|mdx)"],
+  addons: [{
+              name: '@storybook/addon-essentials',
+              options: {
+                toolbars: false,
+              }
+          }, 
+          "storybook-addon-styled-component-theme/dist/register"]
+};
```

Comment:
Formatting is a bit weird here.

Reply:
ESLint / Prettier don't touch these files for some reason? I assume .storybook is ignored somewhere but I can't see where. I have manually adjusted this formatting here